src/authorization.js

```diff
@@ -216,8 +206,8 @@ class Authorization {
    */
   allowsMode (accessMode) {
     // Normalize the access mode
-    accessMode = Authorization.acl[accessMode.toUpperCase()] || accessMode
-    if (accessMode === Authorization.acl.APPEND) {
+    accessMode = acl[accessMode.toUpperCase()] || accessMode
```

Comment:
The JSDoc comment should mention that `accessMode` is either a key that maps to an access mode `NamedNode` or an access mode `NamedNode` itself.